src/object/authenticate.c

```diff
@@ -5860,7 +5860,7 @@ check_authorization (MOP classobj, SM_CLASS * sm_class, DB_AUTH type)
    * Callers generally check Au_disable already to avoid the function call.
    * Check it again to be safe, at this point, it isn't going to add anything.
    */
-  if (Au_disable)
+  if (Au_disable && !(sm_class->flags & SM_CLASSFLAG_SYSTEM))
```

Comment:
Is there any problem in executing db_drop_class() in the au_install() function when executing createdb?

Reply:
I fixed it to createdb not to check autorization. Authorization just check excpets sysadm's altering system class.